src/components/atoms/UiTextarea/UiTextarea.stories.ts

```diff
@@ -71,6 +71,7 @@ export const Basic: TextareaStoryType = {
   },
 };
 Basic.decorators = [ withVModel ];
+Basic.args = { resize: 'vertical' };
```

Comment:
why here? Basic should use 'defaults' as initial args 🙈

Reply:
ok, let we change the default `resize` in UiTextarea to `vertical` if every `textarea` use `vertical` as default :)

Reply:
and remove this changes from UiTextarea stories